code/modules/surgery/organs/robolimbs.dm

```diff
@@ -31,13 +35,15 @@ var/global/datum/robolimb/basic_robolimb
 	company = "Bishop Cybernetics alt."
 	icon = 'icons/mob/human_races/cyberlimbs/bishop/bishop_alt1.dmi'
 	parts = list("head")
+	selectable = 0
```

Comment:
For one, you'd have to build a completely separate list that applied *only* to heads, as the alt heads have sprites that ONLY has that head.

Secondly, the heads make a person look like an IPC. I adhere strongly to TF2's design philosophy of characters when it comes to this area: a distinct character (or in this case species), should be able to be determined at a glance, in a default environment; you shouldn't have to rely on subtleties (or examine) to be able to tell what a species is.

Also it's weird when you have Vulp and Tajarans with IPC heads; their features just float in mid-air.